source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,26 @@
+=====================
+Update Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+You can update multiple documents using the ``updateMany()`` method. 
+``updateMany()`` accepts a filter object, and updates the first document
+that matches the filter using a provided update document. The update
+document requires an :manual:`Update Operator
+</reference/operator/update>` to modify a field in a document.
+
+Create an `Object
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object>`_
+to specify additional options. Set the ``upsert`` option to ``true`` to create a new
+document if no documents match the filter.
+
+The ``updateOne()`` method returns a `Promise
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise>`_
+that resolves to an object. The ``modifiedCount`` field of this object
+shows how many documents were modified. The ``upsertedId`` field contains the ``_id``
+of the new document if a document was ``upserted``.
```

Comment:
upserted probably shouldn't be monospaced here because it's not referencing a value in an object or a variable in code directly, and instead just referring to the concept of upserting.

Reply:
fixed